addons/website/controllers/main.py

```diff
@@ -311,10 +311,13 @@ def get_suggested_link(self, needle, limit=10):
         }
 
     @http.route('/website/snippet/filters', type='json', auth='public', website=True)
-    def get_dynamic_filter(self, filter_id, template_key, limit=None, search_domain=None, with_sample=False):
+    def get_dynamic_filter(self, filter_id, template_key, limit=None, search_domain=None, with_sample=False, **kwargs):
```

Comment:
Actually, this is surely not stable friendly, I spoke a bit too fast yesterday.
If one extended this method and it's now call with this new (unsupported in his override route) params it will crash.
But actually, even if it wasn't I'd go with the context solution (see below) instead so it doesn't need a signature change at all and is encapsulated in the context "feature"